### PR TITLE
PR for home page and navbar

### PR DESCRIPTION
**Pull request by narharikale**
- added navbar component
- added categories section 
- added new arrival section
- done some changes in card component

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well Structures
I liked how you have restructured your products in card and then used them.

**Review by saketkum11 — APPROVED** [review]
Good I like the clean code.

File: src/components/card/card.jsx [hunk 1]
```diff
@@ -1,9 +1,38 @@
-function Card() {
-    return ( 
+
+
+function Card({product}) {
+
+    const { imageSource , name  ,discountedPrice , originalPrice } = product
+
```
Comment (jeetsdev) [inline src/components/card/card.jsx:6]: You can destructure them directly above too like - 
```js
function Card ({ imageSource , name  ,discountedPrice , originalPrice }){
return ()
}
```

File: src/pages/home/home.jsx [hunk 2]
```diff
@@ -21,145 +38,50 @@ function Home() {
...
-                            </a>
-                        </div>
+                        {categories.map(({ id, categoryName, image }) => {
+
+                            return (
+                                <div key={id} className="categories-media">
+                                    <img
+                                        className="responsive-img obj-fit-cover br-top-sm"
+                                        src={image}
+                                        alt="no img"
```
Comment (jeetsdev) [inline src/pages/home/home.jsx:59]: Better to use meaningful alt text.

**Review by jeetsdev — APPROVED** [review]
Added some suggestions, apart from that Code looks clean and well structured 👍🏻

File: src/components/card/card.jsx (hunk 1 above)
Comment (narharikale) [reply src/components/card/card.jsx:6]: I passed the whole obj as props so i destructured product from that  .

File: src/pages/home/home.jsx (hunk 2 above)
Comment (narharikale) [reply src/pages/home/home.jsx:59]: yes fixed

**Review by Mandhularajitha — APPROVED** [review]
clean And Well Structured code

**Pull request merged**